zerver/lib/send_email.py

```diff
@@ -11,6 +11,12 @@
 
 from typing import Any, Dict, Iterable, List, Mapping, Optional, Text
 
+from zerver.lib.logging_util import create_logger
+
+## Logging setup ##
+
+logger = create_logger(__name__, settings.EMAIL_LOG_PATH, 'INFO')
```

Comment:
@hollywoodno, instead of creating the logger here if you create it in the `send_email` function, it should work. Our settings set `disable_existing_loggers = True` so if you create the logger in the function it will be created after the config is initialized. If you want to define the logger here then you need to specify the logger in `settings.LOGGING` as well. Here is the documentation on this setting:
```
f specified as False, loggers which exist when this call is made are left enabled. The default is True because this enables old behaviour in a backward-compatible way. This behaviour is to disable any existing loggers unless they or their ancestors are explicitly named in the logging configuration.
```